src/js/player.js

```diff
@@ -2481,23 +2481,14 @@ class Player extends Component {
 }
 
 /*
- * Global player list
- *
- * @type {Object}
- */
-Player.players = {};
```

Comment:
In 5.0 `videojs` is the main file and everything else gets imported into video.js to be exported. We can't rely on the videojs object being everywhere anymore because then videojs would require everything else and everything else would require videojs. This PR is both adding the circular dependency by importing video.js into player.js and then trying to fix it. Is there some blocker you ran into that sent you down this path?

Things like `videojs.players` and `videojs.options` are just short cuts to lower level lists, so people don't have to do `videojs.getComponent('Player').players` or `videojs.getComponent('Player').prototype.options_`